app/build.gradle

```diff
@@ -79,9 +79,6 @@ android {
 
 def androidSupportVersion = '27.1.1'
 def retrofitVersion = '2.3.0'
```

Comment:
what about those?

Reply:
GSON is also available in a new version and can be updated.

Reply:
Dependabot should handle those.

Reply:
so why do we need to specify the other dependencies individually, but those are OK being specified as variables?

Reply:
Because only firebase and android architecture components announced, that they will no longer simultaneously update their libs.